### PR TITLE
Move HelperText to functional widget

### LINKED ISSUE #926: TextArea, Select, TextInput and Combo box don't pass classes/theme down to HelperText
**Enhancement** 

Currently classes are not passed down in the `HelperText` widget

Package Version: 7.0.0-pre

**Code**

helper-text.tsx

**Expected behavior:**

Should be able to pass classes down from parent widgets:

```
	<TextArea
			key="comment"
			name="comment"
			classes={{
				"@dojo/widgets/helper-text": {
					root: [themedCommonCss.backgroundDarkRed]
				}
			}}
 ...
```

**Actual behavior:**

Classes are not passed down

### PR DESCRIPTION
**Pull request by JamesLMilner**
**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [x] Unit tests are included in the PR
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [x] WidgetProperties are exported

**Description:**

This PR moves `HelperText` to a functional widget and leverages the theme mixin.

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-7qmmzuxto.now.sh
* Tests Deployment: https://dojowidgets-aqrqc7dz8.now.sh

File: src/helper-text/index.tsx [hunk 1]
```diff
@@ -1,42 +1,32 @@
...
+		<div
+			key="root"
+			classes={[
+				themedCss.root,
+				valid === true ? themedCss.valid : null,
+				valid === false ? themedCss.invalid : null
+			]}
+		>
+			{text && (
+				<p key="text" classes={themedCss.text} aria-hidden={'true'} title={text}>
```
Comment (tomdye) [inline src/helper-text/index.tsx]: this can just be `aria-hidden="true"`. Did you get to the bottom of wether or not this is appropriate?

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-a0r6mqblf.now.sh
* Tests Deployment: https://dojowidgets-of5t0kgb1.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-nbmhpmkh4.now.sh
* Tests Deployment: https://dojowidgets-p1d1cepeb.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-nbmhpmkh4.now.sh
* Tests Deployment: https://dojowidgets-p1d1cepeb.now.sh

**Pull request merged**